_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -560,11 +560,18 @@ Testing accounts::
   Vault and Consul clusters after every single commit. You don't want all this infrastructure churn in your application
   or sandbox accounts, so we recommend having a separate AWS account dedicated for automated tests.
 
+Logs account::
+  You will want a single _logs account_ for aggregating log data. All the other accounts—root, security, application
+  accounts, shared-services, etc.—will send their AWS Config and CloudTrail data to this account so that you have a
+  single, central place where all logs are stored and can be viewed. This account will also contain a KMS customer
+  master key (CMK) that is used to encrypt CloudTrail logs.
+
 Note that for larger organizations with multiple separate business units, you may need to repeat the structure above
 multiple times. That is, in the root account, you
 https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_ous.html#create_ou[create an Organization Unit]
-for each business unit, and within each Organization Unit, you create a set of security, application, shared-services,
-sandbox, and testing accounts. It's not unusual for large organizations to have dozens or even hundreds of AWS accounts.
+for each business unit, and within each Organization Unit, you create a set of application, shared-services, sandbox,
+and testing accounts (security and logs accounts are usually shared across the entire organization). It's not unusual
+for large organizations to have dozens or even hundreds of AWS accounts.
```

Comment:
This whole paragraph should be indented at the same level. Otherwise you get line height issues:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/13165182/88315655-2f7fd680-ccdc-11ea-9a3a-9929d78fcfbe.png">

Reply:
This is actually a styling issue on the guides themselves, and not a problem with the content of this particular guide. For some reason, definition lists have more line height than normal paragraph text. So what you're seeing is that the "Logs account" section is part of a definition list, and then the next paragraph ("Note that for larger organizations...") is just a normal paragraph, and when the two are next to each other, it looks weird. We should probably have the same line height on both... Or if anything, _less_ line height on definition lists. It's a minor issue separate from this PR, so CCing @josh-padnick to decide if/when to fix this.

Reply:
FYI, recorded this in https://gruntwork.atlassian.net/browse/WWW-101. Not sure when we'll prioritize this but it's at least recorded for now. 

FWIW, I care a lot about content formatting and consistency. The question is just what's the right place to really invest in that. Right now, I'm planning on coming up with a master docs strategy soon, so that may well be the place we work hard to really get it right.

Reply:
Roger, thx!